### DISCUSSION TITLE
error: self signed certificate with code 'DEPTH_ZERO_SELF_SIGNED_CERT'

### DISCUSSION THREAD

**Discussion by uebmaster** [directus/directus#6220] [Archive]
1) i get error: self signed certificate with code 'DEPTH_ZERO_SELF_SIGNED_CERT' when i try to update from rc.74 to rc.75

2) Exact steps to reproduce this issue
    
inside directus folder
```
pm2 stop 0
npm update
npx directus database migrate :latest
npx directus start
```
    

3) Your environment:
    What version of Directus you are using: **9.0.0-rc.74**
    Which DBMS are you using: **Postgres 12.7**
    Which deployment are you using: **npx (ubuntu server 20.04)**
    What browser are you using: **MS Edge, Chrome, Firefox, Opera, latest versions**

4) Any other relevant information we might need to reproduce this issue
    **NodeJS 14.17.0**

![image](https://user-images.githubusercontent.com/5565506/121735559-e97e5d80-cabb-11eb-9ae4-dd22285fe7ab.png)

**Comment by rijkvanzanten** [comment]
This looks like a configuration issue to me. What database are you using and what configuration do you use to connect?

**Reply by tspvivek** [reply]
I am also getting same issue with rc.75. Downgrading to rc.73 worked for me.

I even tried fresh install with below DB settings. 
My DB settings:
		DB_CLIENT: 'pg',
		DB_HOST: '127.0.0.1',
		DB_PORT: '5432',
		DB_DATABASE: 'directus',
		DB_USER: 'xxxxx',
		DB_PASSWORD: 'xxxxx',
		DB_SSL: 'false',

**Reply by uebmaster** [reply]
sorry, i didn't know that the ticket's template was commented, I have corrected the ticket, now you can see the info about the environment.
@rijkvanzanten

**Reply by rijkvanzanten** [reply]
Ohh I see, there was a bug (that was already fixed) where "false" isn't handled properly in env. Try removing the DB_SSL env var completely 👍🏻

**Reply by uebmaster** [reply]
it works, thanks a lot
